mitmproxy/tools/main.py

```diff
@@ -39,15 +39,10 @@ def process_options(parser, opts, args):
     if args.version:
         print(debug.dump_system_info())
         sys.exit(0)
-    if args.options:
-        print(optmanager.dump(opts))
-        sys.exit(0)
-    if args.quiet:
+    if args.quiet or args.options:
```

Comment:
I expected that dumping options would be done before we'd log anything (just like e.g. `--version`). I can't think of a smart comment either though, so let's go with this.